Lombiq.DataTables/Tests/Lombiq.DataTables.Tests/UnitTests/Services/ExportTests.cs

```diff
@@ -246,4 +240,50 @@ public static IEnumerable<object[]> Data()
             0,
         };
     }
+
+    // Sometimes a font is available, however, it's corrupted or missing a table (for example, this can happen on
+    // GitHub-hosted runners). We can't check directly if a font is missing a table or corrupted, but we can try
+    // other fonts if this happens.
+    private static async Task<Stream> TryExportWithFallbackFontsAsync(
+        ExcelDataTableExportService service,
+        IDataTableDataProvider provider,
+        DataTableDataRequest request,
+        IDictionary<int, string> customNumberFormat)
+    {
+        var fontFamilies = SystemFonts.Collection.Families.ToArray();
+
+        var maxAttempts = Math.Min(3, fontFamilies.Length);
+        var fontIndex = 0;
+
+        while (fontIndex < maxAttempts)
```

Comment:
I know, I'm late to the parts, but together with line 284, this makes for a nice `for` loop 😉